800.renames-and-merges/l.yaml

```diff
@@ -84,6 +84,7 @@
 - { setname: librewolf,                name: [librewolf-app, librewolf-browser] }
 - { setname: libtool,                  name: libtool-bin, addflavor: bin }
 - { setname: license-finder,           name: [licensefinder, ruby:license-finder] }
+- { setname: lidarr,                   name: [$0-develop, $0-develop-bin, $0-nightly-bin], weak_devel: true, nolegacy: true }
```

Comment:
https://repology.org/projects/?search=lidarr
there is no `lidarr-develop-bin` and no `lidarr-nighly-bin`